_inc/client/components/jitm/index.jsx

```diff
@@ -0,0 +1,193 @@
+		this.props.activateModule( module_slug );
+		// to-do: Maybe add tracking here if needed, not sure if activateModule triggers those already.
+		// something like
+		// analytics.tracks.recordEvent( 'jetpack_wpa_module_toggle', {
+		// 	module: module_slug,
+		// 	toggled: 'on'
+		// } );
+	};
+
+	render() {
+		const jitm = this.props.Jitm;
+
+		// to-do: I am not familiar enough with lodash yet, but there may be a way to combine some of those calls.
+		const activate_module = get( jitm, 'activate_module', null );
+
+		const cta = get( jitm, 'CTA', null );
+		const ctaMessage = get( cta, 'message', null );
+		const ctaNewWindow = get( cta, 'newWindow', null );
+		const ctaPrimary = get( cta, 'primary', null );
+
+		const description = get( jitm, 'content.description', null );
+		const feature_class = get( jitm, 'feature_class', '' );
+		const icon = get( jitm, 'content.icon', '' );
+		const id = get( jitm, 'id', '' );
+		const list = get( jitm, 'content.list', null );
+		const mainClasses = classnames(
+			'jitm-card jitm-banner is-upgrade-premium',
+			{ 'has-call-to-action': ctaMessage ? true : false },
+			get( jitm, 'content.classes', '' )
+		);
+		const title = get( jitm, 'content.message', '' );
+		const url = get( jitm, 'url', '' );
+
+		return (
+			<div>
+				{ ! isEmpty( jitm ) &&
+					<div className={ mainClasses }>
+						<div
+							className="jitm-banner__icon-plan"
+							/*eslint-disable react/no-danger*/
+							dangerouslySetInnerHTML={ { __html: icon } }
+						/>
+						<div className="jitm-banner__content">
+							<div className="jitm-banner__info">
+								<div className="jitm-banner__title">{ title }</div>
+								{ description &&
+									<div className="jitm-banner__description">
+										{ description }
+										{ list.length > 0 &&
+											<ul className="banner__list">
+												{ this.renderListItem( list, id, feature_class ) }
+											</ul>
+										}
+									</div>
+								}
+							</div>
+							{ activate_module &&
+								<div className="jitm-banner__action" id="jitm-banner__activate">
+									<Button
+										className="jitm-button"
+										primary={ true }
+										compact={ true }
+										onClick={ this.activateModule( activate_module ) }
+										disabled={ this.props.isActivatingModule( activate_module ) }
+									>
+										{ this.props.isActivatingModule( activate_module ) ? __( 'Activating' ) : __( 'Activate' ) }
+									</Button>
+								</div>
+							}
+							{ ctaMessage &&
+								<div className="jitm-banner__action">
+									<Button
+										// to-do: missing the option to open in a new window with the ctaNewWindow const.
+										// target={ ctaNewWindow === false ? '_self' : '_blank' }
+										href={ url }
+										className="jitm-button"
+										primary={ activate_module === null && ctaPrimary ? true : false }
+										compact={ true }
+										onClick={ this.trackLearnMore( feature_class ) }s
```

Comment:
Tipo here, an extrac `s` at the end of the line.

Symptom was that `Button` gets rendered with and `s` prop.

```
warning.js:33 Warning: Unknown prop `s` on <a> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in a (created by Button)
    in Button (created by Jitm)
    in div (created by Jitm)
```

Reply:
Fixed in 63ad0ec97e39ed443462fa53383fae2d9d46e0f5